csug/system.stex

```diff
@@ -5095,8 +5095,12 @@ is associated with \var{key}.
 environment of the process,
 where it is available to the current process (e.g., via \var{getenv})
 and any spawned processes.
-The key and value are copied into storage allocated outside of
-the Scheme heap; this space is never reclaimed on non-Windows systems.
+There are no particular requirements for the scope or storage of
+arguments however \var{key} should not be empty, {\tt null} or contain
+the character {\tt =}.  \var{value} should not be {\tt null}.  The
+arguments are copied before being incorporated in the environment.
+Therefore their storate is outside of the Scheme heap and the space
+cannot be reclaimed on non-Windows systems.
 
```

Comment:
I'd prefer to simply delete the sentence about storage allocation.

Reply:
Done

Reply:
The inserted lines 5098-5101 are still there.